src/components/SelectComponents/Autosuggest/Autosuggest.tsx

```diff
@@ -30,21 +30,32 @@ interface Props<S>
     inputPlaceholder: string;
     /** String to be shown when no suggestions are available */
     noSuggestionsPlaceholder?: string;
-    /** Defines if the first item of suggestions list is visible even while loading other elements */
-    isFirstItemAlwaysVisible?: boolean;
+    /** Defines if suggestions list is visible even while loading other elements */
+    allowMixingSuggestionsAndLoading?: boolean;
     /** Enable ListOptimizer component for decreasing render time */
     useOptimizeListRender?: boolean;
     /** Function to be called when a suggestion is removed  */
     onSelectionRemove?: (item: S) => void;
     /** Function to be called on submitting form */
     onSubmit?: () => void;
-    /** if suggestions are still loading, i.e. display placeholders */
+    /** If suggestions are still loading, i.e. display placeholders */
     isLoading?: boolean;
+    /** Means to overwrite the look and feel of the UI in its blurred state */
+    customBlurParams?: {
+        /** React node that will be rendered to indicate the current state of selections */
+        selectionIndicator: ReactNode;
+        /** Should the input field be shown. Normally should set this to true (hide) */
+        isInputHidden: boolean;
+        /** Should a clear button be shown on hover */
+        showClearButton: boolean;
```

Comment:
Not really understand why we have `showClearButton` two times in props.

Reply:
These are inside a customBlurParam, so they can use a different use case.

Reply:
It is needed for the logic here: https://github.com/textkernel/oneui/pull/436/files#diff-41965cb515e81f9279b4c06f7984d72b53f05bdbe8bb6c7d4d097cf921465d0fR214